introduction/research-design-principles.qmd

```diff
@@ -23,36 +22,28 @@ Design principles
+When we consider whole designs rather than just thinking about one aspect at a time, we notice how designs that have "parallel" theoretical and empirical sides tend to be strong. We develop this idea in @sec-ch9s3. If you want your estimate $a_{d^*} = A(d^*)$ to be close to the estimand  $a_{m^*} = I(m^*)$, it's often best to choose data strategies that parallel models and answer strategies that parallel inquiries, i.e., to make sure that this rough analogy holds: *M*:*I*::*D*:*A*.
 
 :::
 
-
 ::: {#exm-designagnostically}
-
 Design agnostically
-
-When we design a research study, we have in mind a model of how the world works. But a good design should work, and work well, even when the world is different than what we expect. One implication is that we should entertain many models, seeking not just to ensure the design produces good results for models that we think likely but trying to expand the set of possible models for which the design delivers good results. A second implication is that inquiries and answer strategies should still work when the world looks different to what we expect. Inquiries should have answers even when event generating processes are different to how you imagine them.  In the same way, the ability to apply an answer strategy should depend as little as possible on strong expectations of how the data you will get will look. 
-
+When we design a research study, we have in mind a model of how the world works. But a good design should work, and work well, even when the world is different from what we expect. One implication is that we should entertain many models, seeking not just to ensure the design produces good results for models that we think likely but trying to expand the set of possible models for which the design delivers good results. A second implication is that inquiries and answer strategies should still *work* when the world looks different from what we expect. Inquiries should have answers even when event generating processes are different to how you imagine them.  In the same way, the ability to apply an answer strategy should depend as little as possible on strong expectations of how the data you will get will look. 
 A corollary to "Design agnostically" is that we should know for which models  our design performs well and for which models it performs poorly. We want to diagnose over many models to find where designs break. All designs break under some models, so the fact that a design ever breaks is no criticism. As research designers, we just want to know which models pose problems and which do not.
 
 :::
-
```

Comment:
This should not be deleted, be careful not to delete line breaks around ::: (this happens a few times here)

Reply:
I left line breaks around all :::, hopefully this fixes the issue.